racetime/views/autocomplete.py

```diff
@@ -18,15 +18,28 @@ def clean_term(self, term):
         return str(term).strip()
 
     def get(self, request, *args, **kwargs):
+        # Get terms
         term = self.clean_term(request.GET.get('term', ''))
-
-        results = self.get_results(term)
+        name = self.clean_term(request.GET.get('name', ''))
+        scrim = self.clean_term(request.GET.get('discriminator', ''))
+
+        # Validate the type of search type we are doing. Defaults to term
+        if name and scrim:
+            term = name + "#" + scrim
+        elif name and not scrim:
+            term = name
+        elif scrim and not name:
+            term = "#" + scrim
```

Comment:
I think term should take priority over name and scrim, so can we wrap this section in an `if not term` block?